### PR TITLE
Preregistration Banner

### PR DESCRIPTION
**Pull request by harcod**
Site Banner & Menu>Resources

(and for some reason I could commit code because of 'jest' FAILS.
Fixed one. Skipped the other.

### CODE REVIEW

**Comment by harcod** [comment]
image

image

**Comment by harcod** [comment]
Live Preview at https://covid.harcod.com/PR141/

**Review by johnhawkinson — APPROVED** [review]
Looks good, ran it, passes tests, we should get that out fast!

File: src/components/FilterPanel/index.js [hunk 1]
```diff
@@ -116,10 +116,13 @@ export default function FilterPanelParent({
                                 className={classes.mobileButton}
                                 onClick={() => setFilters(inProgressFilters)}
                                 disabled={
-                                    inProgressFilters.filterByZipCode.zipCode &&
-                                    !isZipValid(
+                                    !!(
```
Comment (tkjones117) [inline src/components/FilterPanel/index.js:119]: @harcod did you mean to do this? idk how this is related to the change at hand

Comment (harcod) [reply src/components/FilterPanel/index.js:119]: (Unrelated but...) Yes.  It was required to pass a jest test.  It wouldn't let me commit!!

Comment (harcod) [reply src/components/FilterPanel/index.js:119]: This change forced it to be a 'boolean' that jest was complaining about.

**Review by tkjones117 — APPROVED** [review]
Approving b/c this is marked as Urgent and Rob will make sure the following get addressed:
- fix the cause of needing to skip the jest test
- `Warning: Each child in a list should have a unique "key" prop.`

**Comment by harcod** [comment]
Added [issue 142](https://github.com/livgust/macovidvaccines.com/issues/142) to address this

> Approving b/c this is marked as Urgent and Rob will make sure the following get addressed:
> 
> * fix the cause of needing to skip the jest test
> * `Warning: Each child in a list should have a unique "key" prop.`

**Pull request merged**